Source/References/ItemsHoldr-0.2.1.ts

```diff
@@ -1392,14 +1392,12 @@ module ItemsHoldr {
          */
         private resetItemsToDefaults(): void {
             this.items = {};
+            this.itemKeys = [];
```

Comment:
Originally it set `itemsKeys` to `Object.keys` of `this.settings.values`, but in the for loop, it calls addItem` which pushes the key to `keyItems` without checking if it exists already so there would have been two of each key by the end of the reset.